Example/tvOSSample/tvOSSample/AuthLoginViewController.swift

```diff
@@ -16,25 +16,22 @@ import UIKit
 import FirebaseAuth
 
 class AuthLoginViewController: UIViewController {
+  override func viewDidLoad() {
+    super.viewDidLoad()
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    // Do any additional setup after loading the view.
+  }
 
-        // Do any additional setup after loading the view.
-    }
+  override func didReceiveMemoryWarning() {
+    super.didReceiveMemoryWarning()
+    // Dispose of any resources that can be recreated.
+  }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
-    }
+  // MARK: - Navigation
 
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
-    }
-    */
+  // // In a storyboard-based application, you will often want to do a little preparation before navigation
+  // override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+  //   // Get the new view controller using segue.destinationViewController.
+  //   // Pass the selected object to the new view controller.
+  // }
```

Comment:
This seems to have formatted poorly, but let's just remove it (I can do so in a follow up PR).

Reply:
This was previously commented out with a single `/*` and a matching `*/`. This ended formatting even worse. I've removed this for now.